Bottom: 54794e65f04c3dfabb311140a589e4bd85447870
Top:    29e41be73cdc36ba24039924a8e8a342edeb010d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-06 15:33:38 +0000

Comment object3d_draw_debug() call during expose


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index b2539ca..7cdb152 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2686,7 +2686,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   glDisable (GL_LIGHTING);
 
   draw_crosshair (Output.fgGC, priv);
-  object3d_draw_debug ();
+//  object3d_draw_debug ();
 
   hidgl_flush_triangles (priv->hidgl);
